tests/OpenCloud/Tests/Database/DatabaseTestCase.php

```diff
@@ -37,4 +55,9 @@ public function setupObjects()
         $this->datastore = $this->service->datastore('10000000-0000-0000-0000-000000000001');
         $this->datastoreVersion = $this->datastore->version('b00000b0-00b0-0b00-00b0-000b000000bb');
     }
+
+    protected function assertCriticalMessageWasLogged()
+    {
+        $this->assertNotEmpty($this->getClient()->getLogger()->getCriticalLogMessage());
```

Comment:
Do you want me to use output buffering or are you okay with the current mechanism?